### PR TITLE
fixing some typos in chapter 3-9

### PR DESCRIPTION
**Pull request by bersling**
- fixing some typos in chapter 3-9
- fix missing closing tags

### CODE REVIEW

File: slides/03_ProjectSetup.html [hunk 1]
```diff
@@ -203,7 +203,7 @@ <h4>Do NOT install project dependencies globally!</h4>
                 </ul>
             </section>
             <section>
-                <h3>Lokale Module</h3>
+                <h3>Locale Module</h3>
```
Comment (joachimprinzbach) [inline slides/03_ProjectSetup.html]: Better: local module, has nothing to do with localization

File: slides/06_Templates_Basics.html [hunk 2]
```diff
@@ -397,7 +397,7 @@ <h3>Task 6.5 - Possible Solutionen</h3>
         <section>
             <section>
                 <h2>Two-way data binding - [()]</h2>
-                <h3>&laquo;banana-in-a-box | Bananenkiste&raquo;</h3>
+                <h3>&laquo;banana-in-a-box | Banana in a box&raquo;</h3>
```
Comment (joachimprinzbach) [inline slides/06_Templates_Basics.html]: Leave it as is please ;-)

File: slides/07_Templates_Advanced.html [hunk 3]
```diff
@@ -363,7 +363,7 @@ <h3>Custom Pipe</h3>
                     </code>
                     <code class="hljs html" data-trim contenteditable>
                         {{'Lorem ipsum dolor' | ellipsis:10}}
-                        <!-- Lorem ipsu...-->
+                        <!-- Lorem ipsum...-->
```
Comment (joachimprinzbach) [inline slides/07_Templates_Advanced.html]: This one is important. ;-) Have a look and think about it :laughing:

**Review by joachimprinzbach — CHANGES_REQUESTED** [review]

**Comment by bersling** [comment]
all bad fixes are fixed now

**Review by joachimprinzbach — APPROVED** [review]

**Pull request merged**